plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduIntegrationDecimalColumns.java

```diff
@@ -65,70 +68,53 @@ public void testCreateTableWithDecimalColumn()
     @Test
     public void testDecimalColumn()
     {
-        assertUpdate("CREATE TABLE IF NOT EXISTS test_decimal (" +
-                "id INT WITH (primary_key=true), " +
-                "col_decimal decimal(10, 6)" +
-                ") WITH (" +
-                " partition_by_hash_columns = ARRAY['id'], " +
-                " partition_by_hash_buckets = 2" +
-                ")");
-
-        assertUpdate("INSERT INTO test_decimal VALUES (0, 0.0), (2, 2.2), (1, 1.1)", 3);
-        assertQuery("SELECT * FROM test_decimal WHERE col_decimal = 1.1", "VALUES (1, 1.1)");
-        assertUpdate("DELETE FROM test_decimal WHERE col_decimal = 1.1", 1);
-
-        assertUpdate("DROP TABLE test_decimal");
+        try (TestTable testTable = new TestTable(
+                new TrinoSqlExecutor(getQueryRunner()),
+                "test_decimal",
+                "(id INT WITH (primary_key=true), col_decimal decimal(10, 6)) " +
+                        "WITH (partition_by_hash_columns = ARRAY['id'], partition_by_hash_buckets = 2)")) {
+            assertUpdate(format("INSERT INTO %s VALUES (0, 0.0), (2, 2.2), (1, 1.1)", testTable.getName()), 3);
```

Comment:
TestTable allows to insert data.

Reply:
But it doesn't validates the number of rows inserted.